test/spec/SearchResults/getFacetValues-facetOrdering.js

```diff
@@ -0,0 +1,1518 @@
+          name: 'Paper',
+          path: 'Paper',
+          count: 48,
+          isRefined: false,
+          exhaustive: true,
+          data: null
+        },
+        {
+          name: 'Musical Instruments',
+          path: 'Musical Instruments',
+          count: 40,
+          isRefined: false,
+          exhaustive: true,
+          data: null
+        },
+        {
+          name: 'Household Essentials',
+          path: 'Household Essentials',
+          count: 32,
+          isRefined: false,
+          exhaustive: true,
+          data: null
+        },
+        {
+          name: 'Office Furniture & Storage',
+          path: 'Office Furniture & Storage',
+          count: 14,
+          isRefined: false,
+          exhaustive: true,
+          data: null
+        },
+        {
+          name: 'Movies & Music',
+          path: 'Movies & Music',
+          count: 13,
+          isRefined: false,
+          exhaustive: true,
+          data: null
+        },
+        {
+          name: 'Magnolia Home Theater',
+          path: 'Magnolia Home Theater',
+          count: 10,
+          isRefined: false,
+          exhaustive: true,
+          data: null
+        },
+        {
+          name: 'Furniture & Decor',
+          path: 'Furniture & Decor',
+          count: 6,
+          isRefined: false,
+          exhaustive: true,
+          data: null
+        },
+        {
+          name: 'Batteries & Power',
+          path: 'Batteries & Power',
+          count: 5,
+          isRefined: false,
+          exhaustive: true,
+          data: null
+        },
+        {
+          name: 'Carfi Instore Only',
+          path: 'Carfi Instore Only',
+          count: 4,
+          isRefined: false,
+          exhaustive: true,
+          data: null
+        },
+        {
+          name: 'Home',
+          path: 'Home',
+          count: 1,
+          isRefined: false,
+          exhaustive: true,
+          data: null
+        },
+        {
+          name: 'Microwaves',
+          path: 'Microwaves',
+          count: 1,
+          isRefined: false,
+          exhaustive: true,
+          data: null
+        }
+      ]
+    };
+
+    expect(facetValues).toEqual(expected);
+  });
+
+  test('root pinned (sortRemainingBy alpha)', function() {
+    var data = require('./getRefinements/hierarchical-cards.json');
+    var order = {
+      renderingContent: {
+        facetOrdering: {
+          values: {
+            'hierarchicalCategories.lvl0': {
+              order: ['Appliances', 'Audio', 'Scanners, Faxes & Copiers'],
+              sortRemainingBy: 'alpha'
+            }
+          }
+        }
+      }
+    };
+    var results = data.content.results.slice();
+    results[0] = Object.assign(order, results[0]);
+
+    var searchParams = new SearchParameters(data.state);
+    var result = new SearchResults(searchParams, results);
+
+    var facetValues = result.getFacetValues('hierarchicalCategories');
+
+    var expected = {
```

Comment:
Even though we could just collapse it, I feel like exporting those variables to a file containing all of them (maybe something like `hierarchical-cards-facetOrdering-results`) could help with the readability.

Reply:
the results are different per test though, so that wouldn't really be a good solution

Reply:
Yes, thinking again about it it would be a complicated-naming JS file with only variables.

Reply:
simplified the data